src/app/pages/home/home.component.ts

```diff
@@ -1,4 +1,7 @@
 import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138913)
```suggestion
import { Router } from "@angular/router";
```